apps/condo/domains/ticket/utils/handlers.js

```diff
@@ -321,6 +336,33 @@ const sendTicketCommentNotifications = async (requestData) => {
             organization: { id: organization.id },
         })
     }
+
+    if (eventTypes[CREATE_RESIDENT_COMMENT_BY_RESIDENT_TYPE]) {
+        const ticketExecutorUserId = get(ticket, 'executor.id')
+        const ticketAssigneeUserId = get(ticket, 'assignee.id')
+        const usersToSendMessage = uniq(compact([ticketExecutorUserId, ticketAssigneeUserId]))
+
+        for (const userId of usersToSendMessage) {
+            await sendMessage(context, {
```

Comment:
Should this action be done at the main thread of the application? Could it be provided to a worker?

Reply:
good take, move it to worker